Examples/Tests/ElectrostaticSphere/analysis_electrostatic_sphere.py

```diff
@@ -0,0 +1,115 @@
+#! /usr/bin/env python
+
+# Copyright 2019-2020 Luca Fedeli, Maxence Thevenet, Remi Lehe
+#
```

Comment:
```suggestion
#
# Copyright 2019-2020 David Bizzozero
```
(unless it's derived from another file and still very similar, then also keep the other names, too please)